orca-core/src/main/java/com/netflix/spinnaker/orca/config/OrcaConfiguration.java

```diff
@@ -132,8 +137,8 @@ UserConfiguredUrlRestrictions userConfiguredUrlRestrictions(
 
   @Bean
   public ContextParameterProcessor contextParameterProcessor(
-      List<ExpressionFunctionProvider> expressionFunctionProviders) {
-    return new ContextParameterProcessor(expressionFunctionProviders);
+      List<ExpressionFunctionProvider> expressionFunctionProviders, PluginManager pluginManager) {
```

Comment:
where is the bean for `PluginManager`, i am confused...

Reply:
oh, does it come from pf4j?

Reply:
I think it comes from `SpinnakerPluginManager` which is configured in Kork